src/basic/unit-file.c

```diff
@@ -473,7 +505,7 @@ int unit_file_build_name_map(
 
                                 r = unit_file_resolve_symlink(lp->root_dir, lp->search_path,
                                                               *dir, dirfd(d), de->d_name,
-                                                              &dst);
+                                                              false, &dst);
```

Comment:
doesn't matter, but maybe prefix "false" with a comment `/* resolve_destination_target= */` to make this easier to grok?